lib/AppInfo/Application.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @copyright Copyright (c) 2020 Vinzenz Rosenkranz <vinzenz.rosenkranz@posteo.de>
+ *
+ * @author Vinzenz Rosenkranz <vinzenz.rosenkranz@posteo.de>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Files3d\AppInfo;
+
+use OCA\Files3d\Listener\LoadFiles3dScript;
+use OCA\Viewer\Event\LoadViewer;
+
+use OCP\AppFramework\App;
+use OCP\EventDispatcher\IEventDispatcher;
+use OCP\Files\IMimeTypeDetector;
+
+class Application extends App {
+
+	const APP_ID = 'files_3d';
+
+	public function __construct() {
```

Comment:
ALso, I was wrong, you cannot do dependency injection in Application.php :)
It only works after the app is initialized, so in other php classes :)

Reply:
So I reverted it back to how it should be

Reply:
Makes sense. I was confused why you changed that part :D